app/templates/main/browse-all.html

```diff
@@ -9,88 +9,91 @@
     "Record total (least)": "records_held-asc"
 } %}
 <p class="govuk-body govuk-body--browse-all-signpost">All available records</p>
-<form action="{{ url_for('main.browse', _anchor='browse-records') }}"
-      method="get">
-    <div class="browse-layout">
-        <!-- SORT -->
-        {% if num_records_found > 0 %}
-            <div class="govuk-form-group sort-container__form">
+<div class="browse-layout">
+    <!-- SORT -->
+    {% if num_records_found > 0 %}
+        <div class="govuk-form-group sort-container__form">
```

Comment:
SORT comment is more appropriate here, rather than outside if the if block, right?